docs/modules/ROOT/pages/api_reference.adoc

```diff
@@ -180,6 +180,91 @@ Example response:
 }
 ----
 
+==== Get relayer status
```

Comment:
Thanks for adding these!

Could you create examples in https://github.com/OpenZeppelin/openzeppelin-relayer-sdk/pulls.

This page is likely to be removed with latest work sai has been doing. Idea is to use openapi file to generate api docs.